Jenkinsfile

```diff
@@ -5,4 +5,7 @@ buildNPM {
   runSonarqube = true
   sonarScanDirs = '.'
   runTest = 'yes'
```

Comment:
IMHO, tests should be run after compiling, if I understand correctly. It's also implemented in the example PR https://github.com/folio-org/ui-plugin-find-user/pull/185

Reply:
https://github.com/folio-org/ui-tags/pull/102